### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by thatoddmailbox** [PlanHubMe/PlanHub#6]
paging @medensqui @nywillb

**Comment by medensqui** [comment]
Ooh that could be a problem... @thatoddmailbox 

Maybe a blueprint with PlanHub written on it? That would look cool

**Comment by thatoddmailbox** [comment]
@medensqui but it needs to be small, and square, so it can fit into places like the favicon or the github org logo.

**Comment by medensqui** [comment]
@thatoddmailbox What about the words PlanHub THEN AFTER THIS THE WORDS ARE UNDERNEATH Planning for the unexpected things with Einstein, Malala, etc. behind them

**Comment by medensqui** [comment]
@thatoddmailbox Btw after things it's images in the background

**Comment by thatoddmailbox** [comment]
@medensqui but we would still need a tiny one to fit in places like favicons and github org logos.

**Comment by medensqui** [comment]
@thatoddmailbox So mini one is C+ Style Aka **Plan**Hub

**Comment by amazansky** [comment]
Did we not already do this?
or should we leave this open?

**Comment by willbarkoff** [comment]
@thatoddmailbox We need a header image for forms and other stuff

**Comment by amazansky** [comment]
Oh okay.

**Comment by suufi** [comment]
Why not do a **PH** or **P**h as a favicon and then **Plan**Hub as a full logo? Surely, PlanHub itself wouldn't squeeze inside a favicon without being visible.

**Comment by thatoddmailbox** [comment]
@OnlyTwentyCharacters That does work, but we already have the blue book icon.
![Current icon](https://planhub.me/images/icon128.png)

**Comment by willweinstein** [comment]
@OnlyTwentyCharacters how did you find us? Do you just randomly browse GitHub?

**Comment by suufi** [comment]
Also, what's up with PlanHub not working on desktop? I receive the webpage not available error on Chrome; yet when I access it on mobile it works.

**Comment by willbarkoff** [comment]
Hi @OnlyTwentyCharacters,
We have never had that error reported before, so it would help you sent screenshots, and your OS and Chrome version (it can be found at chrome://help/)
Thanks!

**Comment by suufi** [comment]
@nywillb William, it's rare to have someone report this because the of OS used. It's POSEmbedded 2007. Not many people get their hands on it. The issue deals with the SSL certificate.

**Comment by thatoddmailbox** [comment]
@OnlyTwentyCharacters 
TL;DR: POSEmbedded 2007 (assuming you refer to [Windows Embedded for Point of Service](https://en.wikipedia.org/wiki/Windows_Embedded_Industry#Releases)) is old. (extended support ends April 12, 2016!) Try another browser and/or upgrade OS.

PlanHub uses CloudFlare, which provides our SSL certificate. (see [SSL Labs report](https://www.ssllabs.com/ssltest/analyze.html?d=planhub.me&s=104.27.131.105&latest) for more info)

I could not find information on POSEmbedded 2007, but [Windows Embedded for Point of Service](https://en.wikipedia.org/wiki/Windows_Embedded_Industry#Releases) seems to be a close match. If this is correct, then the version of Windows Embedded you are using is based off of Windows XP SP2.

Windows XP SP2 and below do not support certificates with SHA256 security methods. For security reasons, CloudFlare signs their certificates with SHA256 (see [this page for more information](https://shaaaaaaaaaaaaa.com/check/planhub.me)). This causes a mismatch between the two, resulting in you not being able to access the site.

You should either try another browser (I think Firefox has its own independent cryptography system, which is not affected by the OS) or upgrade your OS.

**Comment by suufi** [comment]
@thatoddmailbox Yeah. It does work on other browsers/OS, I was just pointing it out. When I checked the console on Chrome, I found a SSL certificate mismatch (which basically means it's not supported).

When I checked the metadata on PlanHub, there was nothing in there that could be used to join the team. Same goes for the mysteriousImage.jpg file (that was made in Photoshop CS6 on OS X).

**Comment by thatoddmailbox** [comment]
@OnlyTwentyCharacters Are you [sure](http://pastebin.com/6QmUFR8d)

**Comment by suufi** [comment]
@thatoddmailbox Oh, I though that was something about the imgsrc. I just scanned through it.

**Comment by amazansky** [comment]
Are you sure?
![screenshot](https://raw.githubusercontent.com/PlanHubMe/PlanHub/master/public/images/mysteriousImage.jpg)
And idk what Antarctica is about...

**Comment by willbarkoff** [comment]
I think it is safe to say this issue is closed.

**Comment by amazansky** [comment]
WHO CHANGED MY COMMENTs

**Comment by thatoddmailbox** [comment]
@RobotXLabs Not me. Ask @nywillb :stuck_out_tongue:

**Comment by amazansky** [comment]
@NYWILLB WHAT HAVE YOU DONE

**Comment by suufi** [comment]
@nywillb @thatoddmailbox Do I **have** to be a Dalton student to be on the team? I decoded every message in the end, you shortened that link which sent me to Dalton's login page (which I couldn't use the _magical phrase_).

**Comment by willweinstein** [comment]
@onlytwentycharacters I though we changed it to just a Google login @thatoddmailbox?

**Comment by suufi** [comment]
@willweinstein @thatoddmailbox You tell me (deleted link because link is super secret)

**Comment by thatoddmailbox** [comment]
![screenshot 2015-12-30 at 8 50 04 pm](https://cloud.githubusercontent.com/assets/1904587/12060138/faa9d78e-af36-11e5-9e63-87f962a6afe4.png)

I removed the "only submit once" option. Try it now?

**Comment by suufi** [comment]
@thatoddmailbox What if I did it for fun AND want to help with PlanHub? ;) Submitted.
